components/sidebar/sidebar_menu/sidebar_menu.tsx

```diff
@@ -159,7 +161,7 @@ export default class SidebarMenu extends React.PureComponent<Props, State> {
                 >
                     <button
                         ref={this.menuButtonRef}
-                        className='SidebarMenu_menuButton'
+                        className={classNames([`SidebarMenu_menuButton ${additionalClass}`])}
```

Comment:
You can just pass `additionalClass` in as a separate argument, and `classNames` will sort out including it if it's actually set. Calling it this way ends up sometimes setting `undefined` as a CSS class like in the snapshot for this file.